eclair-core/src/test/scala/fr/acinq/eclair/integration/basic/ZeroConfAliasIntegrationSpec.scala

```diff
@@ -99,8 +99,8 @@ class ZeroConfAliasIntegrationSpec extends FixtureSpec with IntegrationPatience
     }
 
     eventually {
-      if (bcPublic && deepConfirm) {
-        // if channel bob-carol is public, we wait for alice to learn about it
+    if (bcPublic && deepConfirm) {
+      // if channel bob-carol is public, we wait for alice to learn about it
```

Comment:
nit: indenting

Reply:
Done in [9ca8551](https://github.com/ACINQ/eclair/pull/2322/commits/9ca8551a1d16a2cec2acb1ea46373a3d9266946e).